flocx_market/api/app.py

```diff
@@ -1,20 +1,26 @@
-import pecan
-from pecan import make_app
-from flocx_market import model
-from flocx_market.api import config as api_config
+import os
 
+from flask import Flask
+from flask_restful import Api
+from dotenv import load_dotenv
 
-def get_pecan_config():
-    filename = api_config.__file__.replace('.pyc', '.py')   # get the absolute path of the pecan config.py
-    return pecan.configuration.conf_from_file(filename)
+from flocx_market.api.offer import Offer, OfferList
+from flocx_market.api.bid import Bid, BidList
+from flocx_market.api.contract import Contract, ContractList
 
 
-def setup_app():
-    config = get_pecan_config()
-    model.init_model()
-    app_conf = dict(config.app)
-    app = pecan.make_app(
-        app_conf.pop('root'),
-        logging=getattr(config, 'logging', {}),
-        **app_conf)
+def create_app(app_name):
+
+    app = Flask(app_name)
+    load_dotenv(".env")
+    app.config['SQLALCHEMY_DATABASE_URI'] = os.environ.get('DATABASE_URI')
+    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+    app.config['PROPAGATE_EXCEPTIONS'] = True
+    api = Api(app)
+    api.add_resource(Offer, '/offer', '/offers/<string:marketplace_offer_id>')
+    api.add_resource(OfferList, '/offers')
```

Comment:
There is no `/offers` endpoint in the API (see https://github.com/CCI-MOC/flocx/blob/master/docs/api_marketplace.yaml#L37 and following).